frontend/src/app/shared/profile-about-card/profile-about-card.widget.html

```diff
@@ -0,0 +1,37 @@
+<mat-card appearance="outlined">
```

Comment:
If this widget is not used across modules, we can declare it in the `ProfileModule` instead of the `SharedModule`!